apps/api/src/app/workflows-v2/workflow.controller.ts

```diff
@@ -161,6 +164,23 @@ export class WorkflowController {
     );
   }
 
+  @Get('/:workflowId/step/:stepUuid/metadata')
```

Comment:
we do not support fetching by type now? the client won't be missing the data to create a new step?

Reply:
### it's not a usecase anymore